website/databend/docs/sqlstatement/numeric-functions/pi.md

```diff
@@ -0,0 +1,40 @@
+---
+id: numeric-pi
+title: PI
+---
+
+Returns the value of pi as a floating-point value.
+
+## Syntax
+
+```sql
+PI()
+```
+
+## Arguments
```

Comment:
If no arguments is required, we need to point it or remove this section

Reply:
This is a function template but the arguments is NULL.

Reply:
:ok_hand: